### PR TITLE
Renamed "Api diorama" into "Multiplayer Use Cases" (#MTT-6885)

### PR DESCRIPTION
**Pull request by RikuTheFuffs**
### Description

- Renamed "Api diorama" into "Multiplayer Use Cases"

### Issue Number(s)
- [#MTT-6885](https://jira.unity3d.com/browse/MTT-6885)

### Contribution checklist
 - [x] ~~Tests have been added for the project and/or any internal package~~
 - [x] ~~Release notes have been added to the [project changelog](../CHANGELOG.md) file~~
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] JIRA ticket ID is in the PR title or at least one commit message
 - [x] Include the ticket ID number within the body message of the PR to create a hyperlink

### How to validate

Ignore the changes to the files, as renaming causes all files to be marked as changed.
Do a full text search for "Diorama" (case insensitive) and check that there are 0 references.

### CODE REVIEW

**Review by LPLafontaineB — APPROVED** [review]

File: Basic/MultiplayerUseCases/ProjectSettings/ProjectSettings.asset [hunk 1]
```diff
@@ -158,6 +159,7 @@ PlayerSettings:
     Standalone: com.Unity-Technologies.API-Diorama-Sample
```
Comment (fernando-cortez) [inline Basic/MultiplayerUseCases/ProjectSettings/ProjectSettings.asset:159]: Here's a place with the old name.

**Review by fernando-cortez — APPROVED** [review]
One spot found where to rename to the new name. Otherwise, LGTM!

File: Basic/MultiplayerUseCases/ProjectSettings/ProjectSettings.asset (hunk 1 above)
Comment (RikuTheFuffs) [reply Basic/MultiplayerUseCases/ProjectSettings/ProjectSettings.asset]: Fixed

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize?pullRequest=141) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize?pullRequest=141) before we can accept your contribution.<br/><hr/>**Paolo Abela** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/com.unity.multiplayer.samples.bitesize?pullRequest=141) it.</sub>

**Pull request merged**